types/reactstrap/lib/Popover.d.ts

```diff
@@ -3,23 +3,25 @@ import * as Popper from 'popper.js';
 import { CSSModule } from '../index';
 
 export interface PopoverProps extends React.HTMLAttributes<HTMLElement> {
-  [key: string]: any;
-  isOpen?: boolean;
-  toggle?: () => void;
-  target: string | HTMLElement;
-  container?: string | HTMLElement;
-  boundariesElement?: Popper.Boundary | Element;
-  className?: string;
-  placement?: Popper.Placement;
-  innerClassName?: string;
-  disabled?: boolean;
-  hideArrow?: boolean;
-  placementPrefix?: string;
-  delay?: number | {show: number, hide: number};
-  modifiers?: Popper.Modifiers;
-  cssModule?: CSSModule;
-  fade?: boolean;
-  flip?: boolean;
+    [key: string]: any;
+    isOpen?: boolean;
+
+    toggle?: React.KeyboardEventHandler<any> | React.MouseEventHandler<any>;
```

Comment:
When I quickly skimmed the source, I believe this should be `React.MouseEventHandler<any> | () => void`, with the latter originating from: https://github.com/reactstrap/reactstrap/blob/de5f4822ce7d1505a02c775b7396bb15f0970801/src/TooltipPopoverWrapper.js#L113